xen.spec.in

```diff
@@ -748,61 +694,89 @@ fi
 if [ $1 == 1 -a -f /sbin/grub2-mkconfig ]; then
   if [ -f /boot/grub2/grub.cfg ]; then
     /sbin/grub2-mkconfig -o /boot/grub2/grub.cfg
+    sed -i -e '/insmod module2/d' /boot/grub2/grub.cfg
```

Comment:
Can you find why is it here? Looks like a workaround for `grub2-mkconfig` bug, which should really be fixed there. Otherwise user calling it manually will end up with a different config.

Reply:
Indeed, it's said from commit 6ea38f207f89d2e0c98bffd4b79d7a2f5bd4db6d of upstream, it's a workaround for grub2 issue in dom0. So we should remove that part.